newIDE/app/src/MainFrame/EditorContainers/HomePage/index.js

```diff
@@ -297,14 +301,29 @@ export const HomePage = React.memo<Props>(
                         margin: `0px ${windowWidth === 'small' ? 15 : 35}px`,
                       }}
                     >
-                      <ResponsiveLineStackLayout justifyContent="space-between">
-                        <UserChip
-                          profile={authenticatedUser.profile}
-                          onClick={onOpenProfile}
-                          displayNotificationBadge={hasPendingNotifications(
-                            authenticatedUser
+                      <ResponsiveLineStackLayout
+                        justifyContent="space-between"
+                        alignItems="center"
+                      >
+                        <ResponsiveLineStackLayout
+                          justifyContent="flex-start"
+                          noColumnMargin
+                        >
+                          <UserChip
+                            profile={authenticatedUser.profile}
+                            onClick={onOpenProfile}
+                            displayNotificationBadge={hasPendingNotifications(
+                              authenticatedUser
+                            )}
+                          />
+                          {!electron && !isMobile() && (
+                            <TextButton
+                              label={<Trans>Start in-app onboarding</Trans>}
+                              primary
+                              onClick={onOpenOnboardingDialog}
```

Comment:
should we add an analytics event here to see how many people are actually interested in doing it manually?